### PR TITLE
Feature/change timeout logic

### PR DESCRIPTION
**Pull request by cshcomcom**
Done
- Delete blocking timed out transaction logic (executeTransactionAndAddToPool)
- Use received_at to delete old transactions

### CODE REVIEW

File: tx-pool/index.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ class TransactionPool {
     if (!(tx.address in this.transactions)) {
       this.transactions[tx.address] = [];
     }
+    tx.received_at = Date.now();
```
Comment (platfowner) [inline tx-pool/index.js:51]: I think it's not a good idea to change tx itself?

Comment (platfowner) [reply tx-pool/index.js:51]: Instead, what about adding 'created_at' field here?
https://github.com/ainblockchain/ain-blockchain/blob/develop/tx-pool/transaction.js#L16

Comment (cshcomcom) [reply tx-pool/index.js:51]: Looks good to me! I work it and open this pr again

**Comment by cshcomcom** [comment]
@platfowner All comments are addressed! PTAL

**Review by liayoo — APPROVED** [review]
LGTM, thanks! Please wait for @platfowner's review.

File: node/index.js [hunk 2]
```diff
@@ -234,7 +234,7 @@ class BlockchainNode {
           txList.push(createdTx);
         }
       }
-      return { tx_list: txList };
+      return {tx_list: txList};
```
Comment (platfowner) [inline node/index.js:237]: minor: Personally I see that the previous pattern is more readable.

File: tx-pool/index.js [hunk 3]
```diff
@@ -56,6 +56,7 @@ class TransactionPool {
       timestamp: tx.tx_body.timestamp,
       is_finalized: false,
       finalized_at: -1,
+      tracked_at: Date.now(),
```
Comment (platfowner) [inline tx-pool/index.js:59]: `tracked_at: tx.created_at,` ?

File: tx-pool/transaction.js [hunk 4]
```diff
@@ -1,18 +1,19 @@
 const _ = require('lodash');
 const ainUtil = require('@ainblockchain/ain-util');
 const logger = require('../logger')('TRANSACTION');
-const { WriteDbOperations } = require('../common/constants');
+const {WriteDbOperations} = require('../common/constants');
```
Comment (platfowner) [inline tx-pool/transaction.js:4]: ditto

File: tx-pool/index.js (hunk 3 above)
Comment (cshcomcom) [reply tx-pool/index.js]: Good! Done

**Comment by cshcomcom** [comment]
PTAL! I deleted object-curly-spacing according to the google style guide and also changed lint. I think we have to discuss coding convention and change lint before add object-curly-spacing! @platfowner

**Comment by platfowner** [comment]
@cshcomcom  what about just turning the option off until we get any agreement so that authors can choose depending their own preference?
https://eslint.org/docs/rules/object-curly-spacing#when-not-to-use-it

**Comment by cshcomcom** [comment]
Good idea. I turned off option and reverted it! Done

File: tx-pool/transaction.js [hunk 5]
```diff
@@ -201,7 +202,7 @@ class Transaction {
 
   static isValidFormat(txBody) {
     const sanitized = Transaction.sanitizeTxBody(txBody);
-    return _.isEqual(JSON.parse(JSON.stringify(sanitized)), txBody, { strict: true });
```
Comment (platfowner) [inline tx-pool/transaction.js:204]: ditto

**Review by platfowner — APPROVED** [review]
LGTM
w/ some minor comments.
Thanks for the upgrade!

**Comment by cshcomcom** [comment]
Thanks for the review! Now merging

**Pull request merged**